tachyon/crypto/commitments/vector_commitment_scheme.h

```diff
@@ -72,6 +88,18 @@ class VectorCommitmentScheme {
     return derived->DoCommit(container, result);
   }
 
+  // Commit to |container| and stores the commitment in |batch_commitments_| at
+  // |index| if |batch_mode| is true. Return false if |batch_mode| is false or
+  // the size of |container| doesn't match with the size of parameters.
+  template <typename T = Derived, typename Container,
+            std::enable_if_t<
+                VectorCommitmentSchemeTraits<T>::kSupportsBatchMode>* = nullptr>
+  [[nodiscard]] bool Commit(const Container& container, size_t index) {
+    Derived* derived = static_cast<Derived*>(this);
+    if (!batch_commitment_state_.batch_mode) return false;
```

Comment:
```suggestion
    CHECK(batch_commitment_state_.batch_mode);
```

Reply:
I'll fix ones in "univariate_polynomial_commitment_scheme.h" as well.